src/timeline/js/directives/clip.js

```diff
@@ -208,9 +206,7 @@ App.directive('tlClip', function($timeout){
 						// Preview the right side of the clip
 						scope.PreviewClipFrame(scope.clip.id, new_right);
 					}
-
-				},
-
+				}
```

Comment:
TBH, I don't hate trailing commas on multi-line (multi-_page_) lists like this. It's not illegal, and it saves you when someone adds another member after the last one and _forgets_ to append the comma. But removing it keeps 'em honest, I suppose.

Reply:
@ferdnyc - I'd only leave it there if its very likely we will be adding something soon. IMHO.